src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-view.ts

```diff
@@ -687,6 +737,12 @@ class GmailThreadView {
     return null;
   }
 
+  _findSubjectToolbarElement(): HTMLElement | null {
+    var toolbarContainerElements =
+      this._element.querySelectorAll<HTMLElement>('.bHJ');
+    return toolbarContainerElements[0];
```

Comment:
What's the rationale for querySelectorAll here?

Maybe we just want:

```suggestion
    const toolbarContainerElement =
       this._element.querySelector<HTMLElement>('.bHJ');
     return toolbarContainerElement;
```